### PR TITLE
Adds fallback folder for full framework projects.

### PR DESCRIPTION
**Pull request by livarcocc**
**Customer scenario**

Fallback folders were not being set for full framework projects nor multi-tfm projects. Adds fallback folder for full framework projects by removing the condition on the project being netstandard or netcore. Also, sets RestoreAdditionalProjectFallbackFoldersExcludes that NuGet needs to aggregate over in order to remove fallback folders whenever a 1.x project is used.

**Bugs this fixes:** 

Fixes https://github.com/dotnet/sdk/issues/1389

**Workarounds, if any**

Don't use the offline cache. Instead, we will hit the web during restore for Multi-tfm projects.

**Risk**

Low

**Performance impact**

None.

**Is this a regression from a previous update?**

Yes.

**Root cause analysis:**

We were only setting the property for NuGet in the inner-loop.

**How was the bug found?**

Partner team.

@dotnet/dotnet-cli for review

@srivatsn for approval

### CODE REVIEW

**Comment by srivatsn** [comment]
Don't you need to target the 1.1 branch?

**Comment by livarcocc** [comment]
@srivatsn No, we only have this for 2.0 CLI. For 1.0 packages, VS still carries its own offline feed.

File: test/Microsoft.NET.Restore.Tests/GivenThatWeWantToRestoreProjectsUsingNuGetConfigProperties.cs [hunk 1]
```diff
@@ -24,18 +24,23 @@ public GivenThatWeWantToRestoreProjectsUsingNuGetConfigProperties(ITestOutputHel
...
+        [InlineData("netstandard1.3", "1.3", false)]
         [InlineData("netcoreapp1.0", "1.0", true)]
         [InlineData("netcoreapp1.1", "1.1", true)]
         [InlineData("netstandard2.0", "2.0", false)]
         [InlineData("netcoreapp2.0", "2.0app", false)]
+        [InlineData("net461", "461app", false)]
+        // https://github.com/NuGet/Home/issues/5596 and https://github.com/dotnet/project-system/issues/2605
+        //[InlineData("netcoreapp2.0;net461", "multiTFM20app", false)]
+        //[InlineData("netcoreapp1.0;netcoreapp2.0", "multiTFM1020app", true)]
+        //[InlineData("netcoreapp1.0;net461", "multiTFM1046app", true)]
```
Comment (nguerrera) [inline test/Microsoft.NET.Restore.Tests/GivenThatWeWantToRestoreProjectsUsingNuGetConfigProperties.cs]: These can be enabled now, right?

**Review by nguerrera — APPROVED** [review]

**Comment by nguerrera** [comment]
@MattGertz for approval. The corresponding nuget change was approved.

**Pull request merged**